.changeset/gold-geese-perform.md

```diff
@@ -0,0 +1,5 @@
+---
+'@axis-backstage/plugin-jira-dashboard': major
```

Comment:
I would say that this is a "patch" change? Or at least minor.

Reply:
updated to patch